src/index.ts

```diff
@@ -19,11 +19,17 @@ import {
   ClaimWithChildInstance,
   ClaimWithChildSaleData,
 } from "./types";
+import { chunkedPromiseAll } from "./helpers";
 
 export * from "./types";
 
 const defaultPublicSalePurchaseLimit = 100;
 
+export interface IDWithClaimStatus {
```

Comment:
What type of ID is this? Consider renaming to `DomainIdWithClaimStatus` or `ClaimableDomain`